### DISCUSSION TITLE
posts/how-to-upload-files-in-wordpress-programmatically/

### DISCUSSION THREAD

**Discussion by dcangulo** [dcangulo/davidangulo.xyz#139] [Announcements]
<h1>posts/how-to-upload-files-in-wordpress-programmatically/</h1><a href='https://www.davidangulo.xyz/posts/how-to-upload-files-in-wordpress-programmatically/'>https://www.davidangulo.xyz/posts/how-to-upload-files-in-wordpress-programmatically/</a>

**Comment by dcangulo** [comment]
<i>This comment was originally posted by <a href='https://disqus.com/by/HackSmile/'>Hack Smile (@HackSmile)</a> on 2021-06-26 14:28:53.</i><br><p>is this only on wp admin or even on front end?is this only on wp admin or even on front end?</p>

**Comment by dcangulo** [comment]
<i>This comment was originally posted by <a href='https://disqus.com/by/disqus_K8m6IbLDIK/'>Rohan (@disqus_K8m6IbLDIK)</a> on 2022-09-15 12:01:36.</i><br><p>Will the uploaded files be saved in the library of the website?</p>

**Comment by landolfi-us** [comment]
David, thanks so much for this solution!  It got me on the right track. In my case, I wanted to upload to Media Library so I use media_handle_upload with a combo box selection and the assigned attachmentID to automatically assign categories based on the combo box selection (which is a list of document types).
